lib/timescaledb/database/types.rb

```diff
@@ -1,9 +1,12 @@
 module Timescaledb
   class Database
     module Types
-      # @param [String] interval The interval value
+      # @param [String, Integer] interval The interval value
       # @return [String]
       def interval_to_sql(interval)
+        return 'NULL' if interval.nil?
```

Comment:
Very good! probably we should have some nice docs covering how to use the different interval types later! 🚀

Reply:
Makes sense 💪